lib/tokens/session_token.js

```diff
@@ -24,11 +24,7 @@ module.exports = (log, inherits, Token, config) => {
     this.locale = details.locale || null
     this.mustVerify = !! details.mustVerify || false
 
-    if (details.createdAt > 0) {
-      this.accountCreatedAt = details.createdAt
-    } else {
-      this.accountCreatedAt = null
-    }
+    this.accountCreatedAt = details.accountCreatedAt || null
```

Comment:
Oh, yeah. Actually, we ended up wrong in that previous discussion, it originates from this PR in September 2015:

https://github.com/mozilla/fxa-auth-server/pull/1056

Pretty safe to say we don't need it now I think, maybe we should ditch `accountCreatedAt` entirely?

I'm happy to rip it out in a separate PR if you like although fwiw, I think the above means it will always be set to `null` unless I'm missing something? (so we may as well rip it out rather than change it?)